content/en/docs/tasks/administer-cluster/dns-debugging-resolution.md

```diff
@@ -25,6 +25,12 @@ kube-dns.
 
 {{< codenew file="admin/dns/dnsutils.yaml" >}}
 
+{{< note >}}
+This example creates a pod in the `default` namespace. DNS name resolution for 
+services changes based on namespace of the pod. Review 
```

Comment:
nit: "changes based on namespace of the pod" -> "based on the namespace of the pod" or "based on the pod's namespace"?
Could rephrase:
DNS name resolution for services depends on the namespace of the pod?
For more information, review [DNS for Services and Pods](...)?

Reply:
updated, thanks!